libs/vkd3d/resource.c

```diff
@@ -5899,7 +5902,7 @@ void d3d12_desc_create_sampler_embedded(vkd3d_cpu_descriptor_va_t desc_va,
 }
 
 void d3d12_desc_create_sampler(vkd3d_cpu_descriptor_va_t desc_va,
-        struct d3d12_device *device, const D3D12_SAMPLER_DESC *desc)
+        struct d3d12_device *device, const D3D12_SAMPLER_DESC2 *desc)
```

Comment:
Should probably put the FIXME about ignoring flags here too?

Reply:
Fixed.